web-platform/website/src/components/pages/studies/StudiesTooltip.tsx

```diff
@@ -36,12 +40,19 @@ export function StudiesTooltip({
         </AccordionHeader>
         <AccordionBody accordionId="1">
           <ol className={styles.tooltips}>
-            {strings.sections.map(({ title, text }, i) => (
-              <li key={i}>
-                <h1>{title}</h1>
-                <p>{text}</p>
-              </li>
-            ))}
+            {browserType === BrowserType.Chrome
+              ? strings.sections.map(({ title, text }, i) => (
+                  <li key={i}>
+                    <h1>{title}</h1>
+                    <p>{text}</p>
+                  </li>
+                ))
+              : strings.sections.map(({ title, text, text_fx }, i) => (
+                  <li key={i}>
+                    <h1>{title}</h1>
+                    <p>{i > 0 ? text_fx : text}</p>
+                  </li>
+                ))}
```

Comment:
Can we have a single loop and have strings for chrome / ff instead of two loops? WE can have text.firefox and text.chrome e.g.

Reply:
Yes, very true. Thanks